pkgs/by-name/ka/kanban-bash/package.nix

```diff
@@ -0,0 +1,90 @@
+{
+  lib,
+  gawk,
+  which,
+  gnused,
+  gnugrep,
+  ncurses,
+  coreutils,
+  unixtools,
+  stdenvNoCC,
+  makeWrapper,
+  fetchpatch2,
+  fetchFromGitHub,
+  installShellFiles,
+  writableTmpDirAsHomeHook,
+}:
+stdenvNoCC.mkDerivation {
+  pname = "kanban.bash";
+  version = "0-unstable-2024-10-17";
+  src = fetchFromGitHub {
+    owner = "coderofsalvation";
+    repo = "kanban.bash";
+    rev = "325e2c0c3e75f6bb2106d15107559bc1eea6fd5d";
+    hash = "sha256-OE4kKixNG172yMR5vp5O4STq9fGdJVWLIt7j6deNxhI=";
+  };
+  patches = [
+    # bug fix for linux
+    (fetchpatch2 {
+      url = "https://github.com/coderofsalvation/kanban.bash/pull/45.patch?full_index=1";
+      hash = "sha256-di71KpkdJ7x58W6Yw5y1VXHbdLUUXUcr+V4Cu5JP+eg=";
+    })
+  ];
+  nativeBuildInputs = [
+    installShellFiles
+    makeWrapper
+  ];
+  buildPhase = ''
+    runHook preBuild
+    mkdir -p $out/bin
+    mv kanban $out/bin
+    wrapProgram $out/bin/kanban --set PATH \
+      ${lib.makeBinPath [
+        which
+        gawk
+        gnused
+        gnugrep
+        ncurses # tput
+        unixtools.column
+        unixtools.locale
+        unixtools.more
+        coreutils
+      ]}
```

Comment:
Maybe we want to use resholve instead?

Reply:
is it a good idea? when using it I see /nix/store/n1qgbx4a1gisaglwl3my8q3k7169c47r-python2.7-oildev-unstable-2024-02-26

python2.7?

Reply:
ok tests pass with resholve version